cmd/mdmctl/config.go

```diff
@@ -74,14 +77,51 @@ func (cmd *configCommand) Usage() error {
 	const help = `
 mdmctl config print
 mdmctl config set -h
+mdmctl config switch -h
 `
 	fmt.Println(help)
 	return nil
 }
 
-func setCmd(cfg *ClientConfig, args []string) error {
+func checkForOldConfig() error {
+	configPath, err := namedClientConfigPath("default.json")
+	if err != nil {
+		return err
+	}
+	if _, err := os.Stat(configPath); !os.IsNotExist(err) {
+		fmt.Println("Found old style config. You must migrate it to continue")
+		fmt.Println("Run `mdmctl config migrate -name=myconfig`")
+		os.Exit(1)
+	}
+	return nil
+}
+func migrateServerConfig(configName string) error {
+	configPath, err := namedClientConfigPath("default.json")
+	if err != nil {
+		return err
+	}
+	cfgData, err := ioutil.ReadFile(configPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var serverCfg *ServerConfig
+	err = json.Unmarshal(cfgData, &serverCfg)
+	if err != nil {
+		fmt.Errorf("failed to unmarshal %s : %s", configPath, err)
+	}
+	err = saveServerConfig(serverCfg, configName)
```

Comment:
err is not checked.

Reply:
usually, if the err is the _only_ return, I like to use `if err := f(); err != nil { return err }` style instead of checking the error on a new line.